src/index.js

```diff
@@ -64,13 +64,18 @@ export default class TimeAgo extends Component<DefaultProps, Props, void> {
   timeoutId: ?number;
   isStillMounted: boolean = false;
 
+  // Use Number.isNaN if available, otherwise use polyfill (not available in some browsers inc. IE11)
+  checkIsNaN = (value) => {
+    return (Number.isNaN !== undefined) ? Number.isNaN(value) : typeof value === 'number' && isNaN(value);
+  }
```

Comment:
I'm sorry, but how is the function supposed to work anyway? `isNaN` isn't defined. Please just add that.

Also, I still fail to see why adding a polyfill is a problem for you. I'd rather not bloat the codebase with polyfills for features that modern browsers support anyway. Plus something like Babel should add it for you automatically.